README.md

```diff
@@ -26,9 +26,9 @@ to build and run Objective-C and Java tests.
 
 ### Building and running the mobile example apps
 
-The Android example app can be build with bazel: `bazel build
-//examples:android-app`, and then install to a device with `adb install
-bazel-bin/examples/android-app.apk`
+You need to install the Android SDK 30.0.2 and the NDK 21.4.7075529. You need both `ANDROID_SDK_HOME` and `ANDROID_NDK_HOME` env variables set.
```

Comment:
Is there any reason we require these versions?  I don't remember the example app uses any new features that's only available in these versions.

Reply:
Yeah, the sdk version is hardcoded here:
https://github.com/Snapchat/djinni/blob/c7b0a3978ebe34cc7b5e8323c0ecab5b75017ba7/bzl/android_configure.bzl#L27
The NDK versions here:
https://github.com/Snapchat/djinni/blob/c7b0a3978ebe34cc7b5e8323c0ecab5b75017ba7/bzl/android_configure.bzl#L37C17-L37C17
21.4.7075529 specifically is the latest update to that version.
A later NDK should be backwards compatible, however the bazel version specified to build this project only supports up to version 21, and warns (fiercely) that errors can be introduced by using a later NDK version.

Reply:
I just did a quick test and find that android example build still works if I complete remove the hard coded versions

```
--- a/bzl/android_configure.bzl
+++ b/bzl/android_configure.bzl
@@ -23,8 +23,6 @@ def _android_autoconf_impl(repository_ctx):
     native.android_sdk_repository(
         name="androidsdk",
         path="{}",
-        api_level=30,
-        build_tools_version="30.0.2",
     )
 """.format(sdk_home)
 
@@ -33,8 +31,7 @@ def _android_autoconf_impl(repository_ctx):
         ndk_rule = """
     native.android_ndk_repository(
         name="androidndk",
-        path="{}",
-        api_level=21,
+        path="{}"
     )
 """.format(ndk_home)
```

I think we probably should just remove these as we don't really use anything that requires a specific version.

Reply:
I just think we should avoid putting hard coded versions in here because they needlessly prevent people from building the code with whatever they already have in their system.

Reply:
I agree, that is a better solution.
Let me try that.